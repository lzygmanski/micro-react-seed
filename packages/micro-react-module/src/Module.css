.module {
  margin: 1rem;
}
