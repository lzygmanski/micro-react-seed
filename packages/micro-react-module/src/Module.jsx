import React from 'react';
import './Module.css';

export default function Module() {
  return <h1 className="module">Hello from micro-react-module</h1>;
}
