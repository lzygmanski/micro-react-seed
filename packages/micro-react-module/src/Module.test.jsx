import React from 'react';
import { shallow } from 'enzyme';

import Module from './Module';

describe('Module', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<Module />);
    expect(wrapper).toMatchSnapshot();
  });
});
