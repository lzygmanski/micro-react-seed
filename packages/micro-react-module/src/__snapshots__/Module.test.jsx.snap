// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Module renders correctly 1`] = `
<h1
  className="module"
>
  Hello from micro-react-module
</h1>
`;
